docs/contributor_guide/release.rst

```diff
@@ -0,0 +1,39 @@
+.. release_guide
+
+Release Process
+---------------
+
+This page outlines the process for creating a new Fairlearn release.
+The following steps must be performed on a clone of Fairlearn, not
+a fork.
+
+#. Create an annotated tag using the command at the branch point:
+
+    :code:`git tag -a v<x.y.z> -m "Branch point for v<x.y.z> release"`
+
+#. Push the tag to GitHub:
+
+    :code:`git push origin v<x.y.z>`
+
+#. Create the branch for the release:
+
+    :code:`git checkout -b release/v<x.y.z>`
+
+#. On the release branch:
+
+    #. Update the version in `__init__.py` to `x.y.z`
+    #. Update the version in the ReadMe
+
+#. On the `main` branch:
```

Comment:
same, changes should be PRs

Reply:
It states what the current release is. We can certainly talk about changing that, but this reflects the current process.